README.md

```diff
@@ -143,30 +164,46 @@ Format of Source Catalogs
 The format is white-space delimited ascii. The columns contained 
 depend on the configuration.
 
+You can pass catalogs with an integer source ID in the first column (with 
+sourceid_style="index") or no ID (with sourceid_style="none", in which case the first 
+column is expected to contain ra).
+
 When using point photozs (sigmacrit_style="point") the format is the following
 
 ```
 For shear_style="reduced" (using simple reduced shear style)
-        ra dec g1 g2 source_weight z
+        (id) ra dec g1 g2 source_weight z
```

Comment:
I think it would be better to have new examples rather than complicate this line with the (id)

For someone reading this for the first time it is going to be confusing